### PR TITLE
UTC Timestamp Update

### LINKED ISSUE #150: Refactor all date storage instances to use UTC timestamps
# Background

Dates are currently stored in the database as a string (ie. 4/15/2020). Although this can work, it has potential to cause mishaps with calendar data and reservations.

# Task

 - Update all instances that dates are stored to use UTC timestamps
 - The reservations and listings collections both need to be updated

### LINKED ISSUE #159: Write script to migrate listing availabilities to use UTC timestamps
# Background

Relates to #150

# Task

 - Identify listings that do not yet have UTC timestamps in the availability field
 - Convert string formatted times to UTC timestamps

### PR DESCRIPTION
**Pull request by lullofthesea**
Closes #150 
Closes #159 

Updated to use listingId instead of listing (consistency change).  Fixed all*  areas where dates are used.

* - These were parts that were within the testing bounds.  Create listing / Edit listing -> Create reservation -> Check in / out.  Anything outside of those have not been touched.  I will check for them as this continues.

### CODE REVIEW

**Review by nishantbalaji — APPROVED** [review]
LGTM

**Review by aidenszeto — APPROVED** [review]
Script: Tested with sample listings from prod and dates translate as expected. Love the console coloring too

Infrastructure Changes: Tested end-to-end with exception to reservation routes (blocked by #161) and LGTM - just the two `coords` changes we spoke about on Slack and should be good to go.

We can merge this once the `coords` fixes are pushed and work on #161 a different time. Thank you!

**Pull request merged**
